lib/ansible/modules/database/mysql/mysql_variables.py

```diff
@@ -32,6 +32,16 @@
     description:
     - If set, then sets variable value to this
     type: str
+  mode:
+    description:
+    - C(persist) persists a global system variable to
+      the mysqld-auto.cnf option file in the data directory (the variable will survive service restarts).
+    - C(persist_only) behaves like C(persist) but without setting the global variable runtime value.
+    - Supported from MySQL 8.0.
+    - For more information see U(https://dev.mysql.com/doc/refman/8.0/en/set-variable.html).
+    type: str
+    choices: ['persist', 'persist_only']
```

Comment:
As @flowerysong mentioned on IRC, this should have three choices and a default value.

Reply:
@flowerysong , good job! thanks